tests/out/stop_times.txt

```diff
@@ -0,0 +1,1774 @@
+trip_id,arrival_time,departure_time,stop_id,stop_sequence
+199141,05:00:00,05:00:30,773271753,0
+199141,05:00:00,05:00:30,773271757,1
```

Comment:
@mehdisadeghi for another PR, but I guess that this is a bug here. It says that both stop 0 and 1 are reached at the same time and depart at the same time. At the very least it should arrive after the departure of the previous one. or did I get this wrong?

Reply:
Good catch @carocad! This is indeed a bug, I have made a mistake in my loop while generating this data.

Reply:
@carocad I'm pushing a fix for that right now. It is a tiny.